translations/en-us.json

```diff
@@ -985,7 +985,10 @@
             "assignment": "assignment",
             "group": "group",
             "all": "all",
-            "none": "none"
+            "allow": "allow",
```

Comment:
I think we should capitalize the first word. Also project group or "grouping" as a terminology makes sense to us but I don't think that it would be so relevant or meaningful for the end user.